### ISSUE TITLE
"BatchNorm eval trick" in CBNet question

### ISSUE THREAD

**Issue by zlenyk** [VDIGPKU/CBNetV2#62]
Hi!
I wanted to ask you about specific "eval trick" that you put in here:
https://github.com/VDIGPKU/CBNetV2/blob/72334109d5d0c13ec1e8e8f8651f1a288e681c66/mmdet/models/backbones/cbnet.py#L380
It seems to put all modules in "self.cb_linears" into eval model, which generally sounds like a good idea, however I don't see any such batch-norm modules inside cb_linears.
Am I correct that this effectively does nothing? Or am I missing something?

Thank you!
